src/tests/ftest/dfuse/bash_fd.py

```diff
@@ -98,28 +99,34 @@ def run_bashfd(self, il_lib=None):
         else:
             env_str = ""
 
+        self.log_step('Creating a single pool and container')
         pool = self.get_pool(connect=False)
         container = self.get_container(pool)
-        self.start_dfuse(self.hostlist_clients, pool, container)
 
-        fuse_root_dir = self.dfuse.mount_dir.value
+        self.log_step('Starting dfuse')
+        dfuse = get_dfuse(self, self.hostlist_clients)
+        start_dfuse(self, dfuse, pool, container)
+        fuse_root_dir = dfuse.mount_dir.value
 
-        with open(os.path.join(fuse_root_dir, "bash_fd_inner.sh"), "w") as fd:
+        self.log_step('Setting up the \'bash_fd_inner.sh\' script')
```

Comment:
FYI - you can use either " or ' for strings, but if you want to include one of them in the string then use the other to quote the string and you won't need to escape it.
```suggestion
        self.log_step("Setting up the 'bash_fd_inner.sh' script")
```